.github/workflows/ci-tests.patch.yml

```
@@ -1,45 +1,98 @@
-# Workflow patches for skipping Google Cloud integration test CI when Rust code or dependencies
+# Workflow patches for skipping CI tests when Rust code or dependencies
 # aren't modified in a PR.
-name: Integration Tests on GCP
+name: Run tests
 
 # Run on PRs with unmodified code and dependency files.
 on:
   pull_request:
     paths-ignore:
       # code and tests
-      - '**/*.rs'
+      - "**/*.rs"
       # hard-coded checkpoints and proptest regressions
-      - '**/*.txt'
+      - "**/*.txt"
       # test data snapshots
-      - '**/*.snap'
+      - "**/*.snap"
       # dependencies
-      - '**/Cargo.toml'
-      - '**/Cargo.lock'
+      - "**/Cargo.toml"
+      - "**/Cargo.lock"
       # configuration files
-      - '.cargo/config.toml'
-      - '**/clippy.toml'
+      - ".cargo/config.toml"
+      - "**/clippy.toml"
       # workflow definitions
-      - 'docker/**'
-      - '.dockerignore'
-      - '.github/workflows/ci-unit-tests-docker.yml'
-      - '.github/workflows/sub-deploy-integration-tests-gcp.yml'
-      - '.github/workflows/sub-find-cached-disks.yml'
-      - '.github/workflows/sub-build-docker-image.yml'
-
-# IMPORTANT
-#
-# The job names in `ci-integration-tests-gcp.yml`, `ci-integration-tests-gcp.patch.yml` and
-# `ci-integration-tests-gcp.patch-external.yml` must be kept in sync.
+      - "docker/**"
+      - ".dockerignore"
+      - ".github/workflows/ci-tests.yml"
+      - ".github/workflows/sub-ci-unit-tests-docker.yml"
+      - ".github/workflows/sub-ci-integration-tests-gcp.yml"
+      - ".github/workflows/sub-deploy-integration-tests-gcp.yml"
+      - ".github/workflows/sub-find-cached-disks.yml"
+      - ".github/workflows/sub-build-docker-image.yml"
+
+#! IMPORTANT
+#!
+#! The job names in `sub-ci-unit-tests-docker.yml`, `sub-ci-integration-tests-gcp.yml`,
+#! `ci-tests.patch.yml` and `ci-tests.patch-external.yml` must be kept in sync.
 jobs:
-  # We don't patch the testnet job, because testnet isn't required to merge (it's too unstable)
-  get-available-disks:
-    name: Check if cached state disks exist for Mainnet / Check if cached state disks exist
+  build:
+    name: Build CI Docker / Build CI image
```

I guess we should change this one then as well.

```suggestion
    name: Build CI Docker / Build images
```